sh/tests/array-selector_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash 
```

Comment:
this file is not executable.
Also the test passes with an empty solution

Reply:
The empty solution issue was related to the `exit` in the previous solution version - now it should be fixed